docs/content/concepts/layer/validation/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Well-formed Data Formats and Valid OSCAL
+weight: 50
+description: Explanation of well-formedness, validation, and how they relate to OSCAL
+aliases: []
+suppresstopiclist: true
+---
+
+## Who is this document for?
+
+If you are an [OSCAL tool developer](https://pages.nist.gov/OSCAL/about/stakeholders/tooldevelopers/), you should read this document to understand OSCAL, including core terminology the OSCAL community uses in development and troubleshooting.
+
+If you are not an OSCAL tool developer, this technical information might be useful either if you seek general understanding of OSCAL, or if as a tool *user* you encounter them in your work.
+
+## Why are these concepts important?
+
+It is important that OSCAL tool developers implement software that confirm the JSON, XML, and YAML is well-formed and the [OSCAL document instances](https://pages.nist.gov/OSCAL/concepts/layer/) are valid.
+
+Generally, both "validity" and "well-formedness" define OSCAL, such that a conformant OSCAL application can respect them, and an application that does not is *ipso facto* not an OSCAL application.
```

Comment:
```suggestion
Generally, both "well-formedness" and "validity" of OSCAL instance data determines if a conformant OSCAL application can process this data. In this way, "well-formedness" and "validity" define the boundary between what can be considered OSCAL data or not. The degree to which an application can support well-formed, valid OSCAL data defines how conformant the application is.
```

Reply:
NOTE: `determines`->`determine` since well-formedness and validity mean the verb should be plural. After commit, cleanup, fix that.